tests/test_build.py

```diff
@@ -23,3 +23,33 @@ def test_run_pymodules_install_optional_project_dir(self):
             assert run_pymodules_install(ctx, modules, project_dir) is None
         assert m_info.call_args_list[-1] == mock.call(
             'No Python modules and no setup.py to process, skipping')
+
+    def test_strip_if_debuggable(self):
+        ctx = mock.Mock()
+        ctx.python_recipe.major_minor_version_string = "python3.6"
+        ctx.get_site_packages_dir.return_value = "test-doesntexist"
+        ctx.build_dir = "nonexistant_directory"
+        ctx.archs = ["arm64"]
+        ctx.build_as_debuggable = False
```

Comment:
Minor: This one is not needed right, since you're doing `ctx.build_as_debuggable = True` later on just before calling `run_pymodules_install`